<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsTabWidget</class>
 <widget class="QTabWidget" name="settingsTabWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="locationTab">
   <attribute name="title">
    <string>Location</string>
   </attribute>
   <widget class="QGroupBox" name="downloadLocGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Download Locations</string>
    </property>
    <widget class="QPushButton" name="downloadLocPushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="proxyTab">
   <attribute name="title">
    <string>Proxy</string>
   </attribute>
   <widget class="QTableWidget" name="proxyTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="proxyUploadGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Upload Proxy IP List</string>
    </property>
    <widget class="QCheckBox" name="proxyDeletePrevCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Previous List</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="proxyUploadPushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QComboBox" name="proxyIPVersionComboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="proxyFetchPushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Fetch Online</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
